src/coord/src/catalog.rs

```diff
@@ -2364,11 +2364,11 @@ impl Catalog {
                     builtin_table_updates.push(self.state.pack_role_update(&name, -1));
                     vec![Action::DropRole { name }]
                 }
-                Op::DropComputeInstance { name } => {
+                Op::DropComputeInstance { name, id } => {
```

Comment:
Odd that we need both the `name` and the `id`; think we can change the queries that perform this work to not need the id

Reply:
Just updated to remove the `id` field